### DISCUSSION TITLE
hosts

### DISCUSSION THREAD

**Discussion by liumingan** [MetaCubeX/mihomo#1151] [Q&A]
我设置了hosts：cn.bing.com: bing.com,在终端设备上nslookup cn.bing.com的结果返回的是bing.com的ip，可是用内核测试DNS依然返回的是cn的IP，在终端设备上访问cn.bing.com依然是中国的，没有跳转到bing.com

**Comment by nb5p** [comment]
不明白你说的“内核测试DNS”，但你这样修改不能跳转是正常现象
